app/server/utils/user.py

```diff
@@ -842,14 +843,14 @@ def create_user_without_transfer_account(phone):
     :param phone: string with user's msisdn
     :return: User
     """
-    temporary_first_name = 'Unknown'
-    temporary_last_name = 'Unknown'
+    temporary_first_name = 'Unknown first name'
+    temporary_last_name = 'Unknown last name'
     user = User(first_name=temporary_first_name,
                 last_name=temporary_last_name,
                 phone=phone,
                 registration_method=RegistrationMethodEnum.USSD_SIGNUP)
 
-    organisation = g.active_organisation
+    organisation = Organisation.query.get(2)
```

Comment:
ohh I'm not sure about hardcoding the `2`

Reply:
I've reverted this in this: [Reverts organization to which user's are assigned](https://github.com/GrassrootsEconomics/SempoBlockchain/pull/35/commits/40e86ca6f9e5b0aef048402f99cd4d7028347a0e)